### PR TITLE
update iRO Renewal files

### PR DESCRIPTION
**Pull request by allanon**
- [ ] QA Review

### CODE REVIEW

**Comment by Nipodemos** [comment]
:thinking: 
someone has to test this

**Comment by MonaGan** [comment]
not working.

**Comment by delawr0190** [comment]
Hi @allanon 

I believe that you still have the same issue with 0AF1 that you had in your previous commit.  I'm assuming that your script still needs to be updated.

**Comment by sctnightcore** [comment]
@delawr0190  PR by auto tool .

The recvpacket issue is just a little bit.

**Comment by Nipodemos** [comment]
oh, seems like he fixed the update tool

cool

File: tables/iRO/Restart/shuffle.txt [hunk 1]
```diff
@@ -1,218 +1,54 @@
...
-0178 0862
+# Generated: 2018-08-09T00:00:50Z
+007D 0365
+0090 0A5A
+00B8 0361
+0126 0140
+0127 0364
+0140 07EC
+0178 014D
 0197 0004
```
Comment (c4c1n6kr3m1) [inline tables/iRO/Restart/shuffle.txt]: 0004 - can you re-check this one, the recvpacket is weird , both restart and renewal

**Comment by Rovejoe** [comment]
@delawr0190 can u updates new patch for iro  ?

**Comment by delawr0190** [comment]
@Rovejoe https://github.com/OpenKore/openkore/pull/2236

File: tables/iRO/Renewal/recvpackets.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Generated: 2018-08-03T00:02:13Z
+# Generated: 2018-08-10T02:41:58Z
```
Comment (sctnightcore) [inline tables/iRO/Renewal/recvpackets.txt]: Check : 4e55219

**Comment by mrkye** [comment]
Hi. How I can download all file?

**Pull request closed without merging**